pykickstart/commands/repo.py

```diff
@@ -423,5 +447,22 @@ def _getParser(self):
                     """ % versionToLongString(F27))
         return op
 
+class F30_Repo(F27_Repo):
+    removedKeywords = F27_Repo.removedKeywords
+    removedAttrs = F27_Repo.removedAttrs
+
+    def _getParser(self):
+        op = F27_Repo._getParser(self)
+        op.add_argument("--sslcacert", version=F30, help="""
+                        Path to the directory or file containing the
+                        certificate authorities to verify SSL certificates.""")
+        op.add_argument("--sslclientcert", version=F30, help="""
+                        Path to the SSL client certificate used to connect to
```

Comment:
Here it also might make sense to be more specific what certificate formats are supported & possibly how the path could look like ("file://foo, /foo, foo ?) etc.

Reply:
Agreed with Martin here.  This help string needs to be more explicit.